administrator/components/com_contact/src/Table/ContactTable.php

```diff
@@ -231,6 +231,10 @@ public function check()
             $this->modified_by = $this->created_by;
         }
 
+        if (empty($this->hits)) {
+            $this->hits =0;
```

Comment:
```suggestion
            $this->hits = 0;
```
Fix PHPCS failing due to missing space.